bundles/core/org.eclipse.smarthome.core/src/main/java/org/eclipse/smarthome/core/scheduler/ExpressionThreadPoolManager.java

```diff
@@ -36,18 +35,14 @@
  */
 public class ExpressionThreadPoolManager extends ThreadPoolManager {
 
-    private final static Logger logger = LoggerFactory.getLogger(ExpressionThreadPoolManager.class);
```

Comment:
Instead of using logger for every inner class, you could also do this:

```java
private final Logger logger = LoggerFactory.getLogger(MethodHandles.lookup().lookupClass());
```
This will be automatically resolved for every class instance that uses it.